src/main/java/com/teragrep/cfe18/CaptureMapper.java

```diff
@@ -53,34 +54,34 @@
 
 @Mapper
 public interface CaptureMapper {
-    CaptureFile getCaptureFileById(int id,Integer version);
 
-    CaptureRelp getCaptureRelpById(int id,Integer version);
+    CaptureFile createFile(String tag,
+                        String retention,
+                        String category,
+                        String application,
+                        String index,
+                        String sourcetype,
+                        String protocol,
+                        String flow,
+                        String tagPath,
+                        String capturePath,
+                        int fileProcessingTypeId);
 
-    CaptureFile addNewCaptureFile(String tag,
-                                  String retention,
-                                  String category,
-                                  String application,
-                                  String index,
-                                  String sourcetype,
-                                  String protocol,
-                                  String flow,
-                                  String tag_path,
-                                  String capture_path,
-                                  int processing_type_id);
-
-    CaptureRelp addNewCaptureRelp(String tag,
-                                  String retention,
-                                  String category,
-                                  String application,
-                                  String index,
-                                  String sourcetype,
-                                  String protocol,
-                                  String flow
+    CaptureRelp createRelp(String tag,
```

Comment:
new mapper for relp type, as endpoint is different, please do not reuse the same